packages/amplify-e2e-tests/src/__tests__/auth_6.test.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  initJSProjectWithProfile,
+  getProjectMeta,
+  deleteProject,
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProjectDir,
+  addAuthWithMaxOptions,
+} from 'amplify-e2e-core';
+import _ from 'lodash';
```

Comment:
It will get taken care of by organizing the imports, but remove lodash if not used.

Reply:
updated